app/models/sac_cas/person.rb

```diff
@@ -99,4 +100,12 @@ def backoffice?
   def sac_membership
     @sac_membership ||= People::SacMembership.new(self)
   end
+
+  private
+
+  def check_data_quality
+    return if (People::DataQualityChecker::ATTRIBUTES_TO_CHECK & saved_changes.keys).empty?
+
+    People::DataQualityChecker.new(self)
```

Comment:
Add a method to call, instead of calling the checks on initialization, for example `check_data_quality`